lib/cli.js

```diff
@@ -1,102 +1,4 @@
-/* eslint-disable max-len */
-// Code in this file should work in all conceivably runnable versions of node.
-// A best effort is made to catch syntax errors to give users a good error message if they are using a node version that doesn't allow syntax we are using in other files such as private properties, etc
+const validateEngines = require('./es6/validate-engines.js')
+const cliEntry = require('path').resolve(__dirname, 'cli-entry.js')
```

Comment:
If we're requiring new syntax here, won't we still get syntax errors?

Reply:
Yes, but not until we have attached our own `uncaughtExeception` handler, in which case it will be logged nicely.